### DISCUSSION TITLE
实用主义（Pragmatism）的工程师文化 —— OSCAR 2022 开源文化分会场 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#82] [Announcements]
# 实用主义（Pragmatism）的工程师文化 —— OSCAR 2022 开源文化分会场 - 「开源之道」

https://opensourceway.community/posts/opensource_culture/oscar-2022-pragmatism/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这篇文章好, 开篇凡尔赛:

> ...我们并不会要求主体，而是希望主体能

要求和希望有什么差别? 其实没有, 只是从道义上, 希望站的更高, 
具体到历史事件, 那就是:

- 兄弟们跟我冲
- 弟兄们给我冲

根本原因, 红帽兄也已经揭示了:

>  你描述的,只是我们的日常...

问题就在, RedHat 的日常, 在中国有公司能作到嘛?
比如, 有从 Google 离开的程序猿, 才发现, 除了 google 整个儿世界, 其它没有任何一家公司真正将开发过程中所有环节都工具化了...

从根儿上说, 什么是文化?就是

> 相对固定的区域里
> 相对固定的人群中
> 面对相对固定的问题时
> 产相对固定的第一反应

和社区的 `相对固定` 一个结构, 而作到这种固定的前提是对应区域里,
有足够稳定的社会/共识/资源/空间/机会/....支持, 这么多相对固定;

否则, 一个不知道下个月是否还存在的公司, 不可能允许员工按照

>  只是我们的日常...

那种模式去行事的...

**Reply by lijiangsheng1** [reply]
我们继续探索～ 作为弱者，我做不到要求别人，我只能要求自己。

**Reply by ZoomQuiet** [reply]
#是也乎, (￣▽￣)
既然是弱者, 那么, 和强者最大的区别应该是:

- 我们没有足够的资本来承受失败
- 他们可以调动甚至于不属于自己的资本来支撑自己的探索

那么, 在一个没有开源生存的空间中, 用开源文化来要求自己的唯一可能, 
就是弱者被淘汰, 换上更加弱但是听话的工人..

这样, 开源文化将逐步消亡, 因为理解并愿意秉承的孤勇者们, 始终各自离散在各自生存空间中自我消失了...
